sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -252,16 +252,8 @@ object Klaviyo {
         // Clear profile identifiers from state
         UserInfo.reset()
 
-        // If we had a push token, re-associate it with an anonymous ID, then erase the local copy
```

Comment:
Do the comments above need to be updated? Mainly the note

>      * NOTE: if a push token was registered to the current profile, Klaviyo will disassociate it
>      * from the current profile. Call `setPushToken` again to associate this device to a new profile

Reply:
ah good catch, and maybe there's mention in the readme too

Reply:
thanks, fixed, and no change was needed in the readme